k8s_pod_tracker_test.go

```diff
@@ -0,0 +1,89 @@
+package k8swatcher
+
+import (
+	"net"
+	"reflect"
+	"testing"
+
+	k8score "k8s.io/api/core/v1"
+)
+
+func TestPodTrackerCreateDelete(t *testing.T) {
+	type podInfo struct {
+		name, ip string
+		labels   map[string]string
+		ready    bool
+		phase    k8score.PodPhase
+	}
+	for _, itbl := range []struct {
+		name         string
+		pods         []podInfo
+		surviving    []string
+		expectedDead map[string]struct{}
+	}{
+		{
+			name: "one_ready",
+			pods: []podInfo{podInfo{name: "foobar", ip: "10.42.42.42", labels: map[string]string{"app": "fimbat"},
```

Comment:
nit: Maybe run `go fmt -s` on this file to remove the unnecessary type annotations...

Reply:
good call.
Done.